posthog/models/session_recording_event/util.py

```diff
@@ -52,6 +57,54 @@ def create_session_recording_event(
     return str(uuid)
 
 
+def bulk_create_session_recording_event(events: List[Dict[str, Any]]) -> None:
+    """
+    Test only
+    """
+    # timestamp = cast_timestamp_or_now(timestamp)
+
+    inserts = []
+    params: Dict[str, Any] = {}
+    for index, event in enumerate(events):
+        timestamp = event["timestamp"]
+        if isinstance(timestamp, str):
+            timestamp = isoparse(timestamp)
+        else:
+            timestamp = timestamp.astimezone(pytz.utc)
+        timestamp = timestamp.strftime("%Y-%m-%d %H:%M:%S.%f")
+        data = {
+            "uuid": str(event["uuid"]),
+            "team_id": event["team_id"],
+            "distinct_id": event["distinct_id"],
+            "session_id": event["session_id"],
+            "window_id": event.get("window_id"),
+            "snapshot_data": json.dumps(event.get("snapshot_data", {})),
+            "timestamp": timestamp,
+            "created_at": timestamp,
+        }
+        inserts.append(
+            """(
+
+                %(uuid_{i})s,
+                %(timestamp_{i})s,
+                %(team_id_{i})s,
+                %(distinct_id_{i})s,
+                %(session_id_{i})s,
+                %(window_id_{i})s,
+                %(snapshot_data_{i})s,
+                %(created_at_{i})s,
+                now(),
+                0
+            )""".format(
+                i=index
+            )
+        )
+
+        params = {**params, **{"{}_{}".format(key, index): value for key, value in data.items()}}
```

Comment:
`params.update()` would make this a bit more clear I think.

Reply:
I'll treat this as a NIT and ignore